src/components/Sum.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({
+  a,
+  b,
+}) => (
```

Comment:
```suggestion
const Sum = ({ a, b, }) => (
```
Не стоит переносить в данном случае